models/models.go

```diff
@@ -76,6 +77,11 @@ func Find(data map[string]interface{}, structure interface{}) error {
 	if !tableNameExists || !columnNameExists || !columnValueExists {
 		return errors.New("missing required keys any of this tablename,columnname,columnvalue")
 	}
+	//checking the columnName provided is matching to the database columnName or not
+	_, exist := ColumnMap[columnName]
+	if !exist {
+		return errors.New("columnn does not exist .Please check the columnname")
```

Comment:
Doesn't inform where to check, should the dev check their model struct or the DB ?

Reply:
Developer didnt need to do anything if the column name doesnt exisit in the db it will return error in the dev controller part